api/google-manager/services/google-manager.js

```diff
@@ -128,8 +155,20 @@ if (!isDevEnv()) {
   const privateKey = googleKey.private_key;
   const groupID = process.env.DEVLAUNCHERS_GOOGLE_DIRECTORY_GROUP_ID;
   const serverBaseURL = process.env.URL;
+  const auditFreq = process.env.AUDIT_FREQ_MINUTES;
+
+  const manager = new GoogleManager(email, privateKey, groupID, serverBaseURL, auditFreq);
+  const cronTime = `0 */${auditFreq} * * * *`;
+  const job = new CronJob(cronTime, function () {
+    manager.listAuditReports('all', 'meet', 'call_ended').then(reports => {
+      reports.items.map(r => {
```

Comment:
This might be a bit nit-picky. I see that you're using map method to loup through items and log reports. You usually use the map method when you want to create a new array with the results of the function that you called on every element. For example, 
```js
const nums = [1,2,3];
const newNums = nums.map(num => {
   return num * 2
})

//returns
[2,4,6]
```
This will return a new array with every element inside of it multiplied by 2. You probably want to be using the forEach method instead because you aren't using the return of map. You're just using it to execute a function on every element of the array which forEach is for. Unless I'm being very dumb and you are using the return of map somewhere lol

Reply:
That's a great suggestion. Right now I'm only logging to see what data is returned, but one day we will consume the array.